src/osversion/VersionTranslator.cpp

```diff
@@ -9160,6 +9160,32 @@ namespace osversion {
         ss << newObject;
         m_refactored.emplace_back(std::move(object), std::move(newObject));
 
+      } else if (iddname == "OS:Schedule:Day") {
+
+        // 1 Field has been added from 3.7.0 to 3.8.0:
+        // ----------------------------------------------
+        // * Interpolate to Timestep * 3
```

Comment:
pedantic man to the rescue!

```suggestion
        // 1 Field has been modified from 3.7.0 to 3.8.0:
        // ----------------------------------------------
        // * Interpolate to Timestep * 3 - Changed from bool to string choice
```

Reply:
Anyways, VT + VT test, :+1: